hack/machineset.sh

```diff
@@ -185,7 +185,7 @@ get_vsphere_ms() {
 
   # set golden image template name
   # TODO: read from parameter
-  template="windows-golden-images/windows-server-2004-template"
+  template="windows-golden-images/windows-server-2022-template"
```

Comment:
We need to update it [here](https://github.com/openshift/release/pull/24911/files#diff-c720c0075993159417e4f67b269d3d26696299d9db2409d49d0928ce660ee6d3R14) too, when https://github.com/openshift/release/pull/24911 merges.

Reply:
True, noted :writing_hand:

Reply:
Thanks for pointing out @jrvaldes . do we have jira ticket to track this?

Reply:
@selansen added to acceptance criteria of https://issues.redhat.com/browse/WINC-602

Reply:
PR opened here: https://github.com/openshift/release/pull/26435